src/som/interpreter/Frame.java

```diff
@@ -227,6 +227,19 @@ public void printStackTrace(final SAbstractObject nilObject) {
     }
   }
 
+  public void popArgumentsAndPushResult(final SAbstractObject result, SMethod method) {
+    // compute the number of arguments
```

Comment:
These type of comments seem to be redundant with the code and don't give a rational.
So, whenever I edit these types of methods, I tent to remove them.

Though, you are having a fresh look at the code, do you find them useful?

Reply:
No, indeed. At least not here. I've removed them